clang/tools/dpct/cmake/dpct.cmake

```diff
@@ -74,3 +74,6 @@ endmacro()
 macro(DPCT_COMPILE_SYCL_CODE generated_files)
   DPCT_COMPILE_SYCL_CODE_IMP(sycl_device ${generated_files} ${ARGN})
 endmacro()
+
+# Always set SYCL_HAS_FP16 to true to assume SYCL device to support float16
```

Comment:
please emit a message like:  "dpct.cmake: SYCL_HAS_FP16 is set true by default."